packages/auth/android/src/main/java/io/invertase/firebase/auth/ReactNativeFirebaseAuthModule.java

```diff
@@ -891,8 +894,13 @@ public void onCodeSent(
         String verificationId,
         PhoneAuthProvider.ForceResendingToken forceResendingToken
       ) {
-        // TODO: This isn't being saved anywhere if the activity gets restarted when going to the SMS app
+        // kyun: This isn't being saved anywhere if the activity gets restarted when going to the SMS app
+        /** Save mVerificationId to local in case the activity gets restarted when going to the SMS app.
+         * So that the app doesn't crash when function _confirmVerificationCode call and can verify normal when using local verificationId
```

Comment:
```suggestion
         * So that the app doesn't crash when function _confirmVerificationCode is called and 
         * it can verify normally by using local verificationId
```